auth/auth_user.py

```diff
@@ -1,17 +1,23 @@
-from pieces_os_client import UserProfile
-
+from pieces_os_client import UserProfile,AllocationsApi
+from pieces.settings import PiecesSettings
 import sublime
 
 class AuthUser:
 	output_panel = None
 	phantom_set = None
+	connect = False # should we connect to the allocation server
```

Comment:
this should not be a necessary property in this class also it's good to avoid the usage of flags like this where possible